main.py

```diff
@@ -27,22 +54,133 @@ def SEARCH():
     key_idx=random.randint(1,25) #randomize box with key
     turncount=0 #to check turn count
     haswon=False #to check if successfully selected the key
+    buttonWidth=240
+    buttonHeight=50
+    buttonLeft=width/2-(buttonWidth/2)
+    buttonRight=width/2+(buttonWidth/2)
 
     done=False #loop variable
 
+    howto = False #Instruction Menu
+    start = False #Game Started
+
+    #Initialise fonts
+    pygame.font.init()
+
     clickedbox=0 #to set default value
-    while not done:
+
+    #Initial Event
+    event = pygame.event.poll()
+
+    while event.type != pygame.QUIT:
+
+        windowSurface.blit(pygame.transform.scale(BG,(size)),(0,0))
+
+        event = pygame.event.poll()
+
+        if event.type == pygame.MOUSEMOTION:
+            x, y = event.pos
+
+        if start == False and howto == False:
+            if event.type == pygame.MOUSEBUTTONDOWN and event.button == 1:
+                if howtoRect.collidepoint(event.pos):
+                    howto = True
+
+                elif startRect.collidepoint(event.pos):
+                    start = True
+                    break
+
+            title = pygame.font.SysFont("symbola",100)
+            textSurf1, textRect1 = text_objects("Find The Key!", title)
+            textRect1.center = ( (width/2), (height/2) - 100)
+
+            findkey = pygame.font.SysFont("liberationserif",20)
+            textSurf2, textRect2 = text_objects("LET'S FIND THE KEY!", findkey)
+            textRect2.center = ( (width/2), (height/2) + 150)
+
+            howtotext = pygame.font.SysFont("liberationserif",20)
+            textSurf3, textRect3 = text_objects("HOW-TO-PLAY", howtotext)
+            textRect3.center = ( (width/2), (height/2) + 225)
+            
+            windowSurface.blit(pygame.transform.scale(STARTGOLDKEY,(200,100)),(412,70))
+
+            if x>=buttonLeft and x<=buttonRight and y>=height/2+125 and y<=height/2+125+buttonHeight:
+                pygame.draw.rect(windowSurface, GOLD, startRect)
+
+            else:
+                pygame.draw.rect(windowSurface, GREEN, startRect)
+
+
+            if x>=buttonLeft and x<=buttonRight and y>=height/2+200 and y<=height/2+200+buttonHeight:
+                pygame.draw.rect(windowSurface, GOLD, howtoRect)
+
+            else:
+                print x,y

```

Comment:
@ckarthik14 these were unnecessary prints you probably forgot to remove before sending the PR. I've removed it for now but make sure next time you remove debug print statements, whitespaces etc.

Reply:
@vineetjc yes, I used them for debugging. I'll remember to remove them next time.